.github/workflows/lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Python application
```

Comment:
I'm confused by why this is part of release maker? Is it assuming release maker will only run on Python repos?

Reply:
This isn't part of release maker. It's part of the repository because the release maker is a python application. It's like how we added black/flake to our test scripts for other repos.